src/app/components/Text/__snapshots__/index.test.jsx.snap

```diff
@@ -2,15 +2,41 @@
 
 exports[`Text with data should render correctly 1`] = `
 Array [
-  <p>
-    This is a paragraph.
-</p>,
-  <p>
-    This is another paragraph.
-</p>,
-  <p>
-    This is a final paragraph.
-</p>,
+  <div>
+    <div>
+        <p>
+            It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George's Chapel and St George's Hall using locally sourced foliage, 
+            <a
+                href="/news/articles/c000000000ro"
+            >
+                which were later donated to local hospices
+            </a>
+            .
+        </p>
+    </div>
+</div>,
+  <div>
+    <div>
+        <p>
+            This is another paragraph with some 
+            <strong>
```

Comment:
@hindsc52 FYI we are going with an italic and bold tag, rather than em and strong to align semantically with editorial use cases

Reply:
@greenc05 That is fine, this PR simply introduces the ability to parse the markdown from the text blocks.

Once this gets merged we will be in a position to create customs stateless components to render bold, italic and link elements. It is in these custom components where any extra UX and accessibility requirements should be met.

This is just laying the ground work.

Reply:
Fabulous @hindsc52